gradle/versions.gradle

```diff
@@ -2,15 +2,15 @@
 // Taken from android-complete.
 ext {
     // SDK
-    minSdkVersion = 16
+    minSdkVersion = 21
```

Comment:
I guess this was missed because we never merged it to ADAL, so having it here is fine